File: Python/SelectionSort.py
def SelectionSort(arr):
    for i in range(len(arr)-1):
        key=arr[i]
        index=i
        for j in range(i+1,len(arr)):
            if (arr[j]<key):
                key=arr[j]
                index=j
        temp=arr[i]
        arr[i]=arr[index]
        arr[index]=temp
    return


mylist=[123,23,4,1,23,34]
SelectionSort(mylist)
print(mylist)

File: Javascript/SelectionSort.js
var arr = [123,45,0,-987,876,1234,1,2,3,56];

function SelectionSort(arr)
{
    for(var i=0;i<arr.length-1;i++)
    {
        var index = i;
        for(var j=i+1;j<arr.length;j++)
        {
            if(arr[j]<arr[index])
            {
                index = j;
            }
        }

        var temp = arr[index];
        arr[index] = arr[i];
        arr[i] = temp;
    }
    return arr;
}

var result = SelectionSort(arr);
console.log("This is selection sort procedure");
console.log(result);